src/jllvm/llvm/ClassObjectStubImportPass.cpp

```diff
@@ -59,7 +59,7 @@ llvm::PreservedAnalyses jllvm::ClassObjectStubImportPass::run(llvm::Module& modu
             },
             [&](const DemangledStaticCall& staticCall) -> llvm::Function*
             {
-                ClassObject* classObject = m_classLoader.forNameLoaded(staticCall.classDescriptor);
+                ClassObject* classObject = m_classLoader.forNameLoaded(ObjectType(staticCall.className));
```

Comment:
`FieldType::fromPretty` should be required here and below as well, just like in the DefinitionsGenerator